sap_hana/datadog_checks/sap_hana/data/conf.yaml.example

```diff
@@ -11,17 +14,26 @@ init_config:
     #     columns: <COLUMNS>
     #     tags: <TAGS>
 
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
 instances:
 
     ## @param server - string - required
     ## The hostname used to connect to the system.
     #
   - server: <SERVER>
 
-    ## @param port - integer - required - default: 39015
```

Comment:
The default port is 30015 (see https://github.com/DataDog/integrations-core/blob/ff62dcf800dad92a2710d2cf59f72516a5c51075/sap_hana/datadog_checks/sap_hana/sap_hana.py#L33)

Reply:
@ofek Seems both port are valid, 39015 being "HXE2" and 30015 for "HXE1".
Any idea which one we should default to?